src/components/AppNavigationSettings/AppNavigationSettings.vue

```diff
@@ -26,10 +26,11 @@
 		:class="{ open }">
 		<div id="app-settings-header">
 			<button
-				class="settings-button"
+				class="settings-button-vue"
```

Comment:
We could take that opportunity to drop the server styling and move to proper class BEM.

```suggestion
		<div class="app-settings">
			<button
				class="app-settings__button"
```

Reply:
Done.

Reply:
I meant the entire app-settings :)
But that could be another PR

Reply:
> 
> 
> I meant the entire app-settings :)
> But that could be another PR

I guess, I didn't understand what you mean. 🙈